apps/ff-site/src/app/_components/BreadCrumbsLayout.tsx

```diff
@@ -1,13 +1,30 @@
-import React from 'react'
+'use client'
 
-import { BreadCrumbs } from '@/components/BreadCrumbs'
+import Link from 'next/link'
+import { usePathname } from 'next/navigation'
+
+import { BreadCrumbs } from '@filecoin-foundation/ui/BreadCrumbs'
+import type { Route } from 'next'
+
+import { PATHS } from '@/constants/paths'
 
 import type { LayoutProps } from '@/layout'
 
 export function BreadCrumbsLayout({ children }: LayoutProps) {
+  const pathname = usePathname()
+
   return (
     <div className="flex grow flex-col gap-8">
-      <BreadCrumbs />
+      <BreadCrumbs
+        currentPath={pathname}
+        homePath={PATHS.HOME.path}
+        homeLabel={PATHS.HOME.label}
```

Comment:
Can we just use `/` and `Home`?

```suggestion
        homePath={PATHS.HOME.path}
        homeLabel={PATHS.HOME.label}
```